File: Assignment8/program5.c
//Write a program which accept number from user and display its table in reverse order

//4. Write a program which accept number from user and display its talble

#include<stdio.h>

void TableRev(int iNo)
{
    int iCnt = 0;
    for(iCnt = 10; iCnt>=1; iCnt--)
    {
        printf("%d\t", iNo*iCnt);
    }
}
//Time Complexity: O(10)
int main()
{
    int iValue = 0;

    printf("Enter Number: ");
    scanf("%d", &iValue);

    TableRev(iValue);

    return 0;
}

File: Assignment17/Program5.java
/* Write a program to print the multiplication table of a number.*/
package Assignment17;

class Logic
{
    void printTable(int num)
    {
        int iCnt = 0;
        for(iCnt = 1 ; iCnt<=10 ; iCnt++)
        {
            System.out.println(num * iCnt);
        }
    }
}

class Program5 
{
    public static void main(String A[])
    {
        Logic obj = new Logic();
        obj.printTable(5);
    }
}
